src/day8/re_frame/trace/app_state.cljs

```diff
@@ -0,0 +1,42 @@
+(ns day8.re-frame.trace.app-state
+  (:require [reagent.core :as r]
+            [clojure.string :as str]
+
+            [cljs.pprint :refer [pprint]]))
+
+
+(defn css-munge
```

Comment:
It's good to split these functions out. I'd recommend making a short function name which combines them both so you can use it everywhere consistently, and make it easier to see what's going on.